Cheetah/legacy_compiler.py

```diff
@@ -339,7 +339,9 @@ def _addAutoSetupCode(self):
         self.indent()
         self.addChunk('_dummyTrans = False')
         self.dedent()
-        self.addChunk('SL = self._CHEETAH__searchList')
+        self.addChunk('locals_ = locals()')
+        self.addChunk('globals_ = globals()')
```

Comment:
yeah, but we don't have that information at this point.  Can be a followup when we have a better parse tree